File: Sheet #1 (Data type - Conditions)/C++/S-Interval.cpp
#include <iostream>
using namespace std;

int main()
{
    double num;

    cin >> num;

    if (num < 0 || num > 100)
    {
        cout << "Out of Intervals" << endl;
        return (0);
    }

    if (num <= 25 && num >= 0)
        cout << "Interval [0,25]" << endl;

    else if (num <= 50)
        cout << "Interval (25,50]" << endl;

    else if (num <= 75)
        cout << "Interval (50,75]" << endl;

    else if (num <= 100)
        cout << "Interval (75,100]" << endl;

    return (0);
}

File: Sheet #1 (Data type - Conditions)/python/S-Interval.py
#!/usr/bin/python3

"""
Given a number X.
Determine in which of the following intervals
the number X belongs to:
[0,25], (25,50], (50,75], (75,100]
"""

value = float(input(""))
if value < 0 or value > 100:
    print("Out of Intervals")

elif value <= 25:
    print("Interval [0,25]")

elif value <= 50:
    print("Interval (25,50]")

elif value <= 75:
    print("Interval (50,75]")

elif value <= 100:
    print("Interval (75,100]")
